src/lib.rs

```diff
@@ -228,6 +229,7 @@
 #![allow(non_camel_case_types, non_snake_case,
         // FIXME: these types are unsound in C FFI already
         // See https://github.com/rust-lang/rust/issues/53346
+        incomplete_features,
```

Comment:
nit: The FIXME comment should be above on improper_ctypes_definitions.

Reply:
insert joke about const generics also being difficult to use in C FFI due to monomorphizing.
good catch! thank you.